shared/src/iosMain/kotlin/com/fbada006/shared/utils/ShareableContent.ios.kt

```diff
@@ -0,0 +1,76 @@
+package com.fbada006.shared.utils
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.LaunchedEffect
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.asComposeImageBitmap
+import androidx.compose.ui.interop.LocalUIViewController
+import com.fbada006.shared.actions.ArtMakerAction
+import kotlinx.cinterop.ExperimentalForeignApi
+import kotlinx.cinterop.useContents
+import org.jetbrains.skia.Bitmap
+import org.jetbrains.skia.ColorAlphaType
+import org.jetbrains.skia.ColorType
+import org.jetbrains.skia.ImageInfo
+import platform.CoreGraphics.CGSizeMake
+import platform.UIKit.UIGraphicsBeginImageContextWithOptions
+import platform.UIKit.UIGraphicsEndImageContext
+import platform.UIKit.UIGraphicsGetImageFromCurrentImageContext
+
+@OptIn(ExperimentalForeignApi::class)
+@Composable
+actual fun ShareableContent(
+    shouldExport: Boolean,
+    onAction: (ArtMakerAction) -> Unit,
+    content: @Composable () -> Unit,
+    modifier: Modifier
+) {
+    val uiViewController = LocalUIViewController.current
+
+    LaunchedEffect(shouldExport) {
+            val view = uiViewController.view
```

Comment:
You should add a check for `if (shouldExport)` inside the launched effect, like in android. Even if it does not work, there is no need to have unnecessary work.